aQute.libg/src/aQute/lib/startlevel/StartLevelRuntimeHandler.java

```diff
@@ -157,16 +157,28 @@ public void close() {}
 	static public StartLevelRuntimeHandler create(Trace logger, Map<String, String> outerConfiguration) {
 
 		String defaultStartlevelString = outerConfiguration.get(LAUNCH_STARTLEVEL_DEFAULT);
-		if (defaultStartlevelString == null) {
+		if (defaultStartlevelString == null || defaultStartlevelString.equals("0")) {
```

Comment:
After the null check, you should `toInt` the value to handle trimming and equivalence ("+0"). A simple "0" equals test is not robust enough.

Reply:
the trimming is already done in bnd, who sets this property. But I can change it.